docs/guide/config.qmd

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Configuring the checks"
+description: "Customise the types of checks that are done against your `datapackage.json` by using the `Config` class, such as excluding certain checks or adding your own."
```

Comment:
It would look aesthetically pleasing if each of these descriptions were written in the same format. The other two you edited started in the same manner: "A brief walkthrough..." and "A short description", whereas this one and the installation one follows a different format starting with a verb and including more of the content in the description. I don't have a strong preference for which way these are written, but think it would look nice if they were more similar.